File: CarlaErik/UvA.SoftCon.Questionnaire/Runtime/Evaluation/Types/StringValue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UvA.SoftCon.Questionnaire.Runtime.Evaluation.Types
{
    public class StringValue : Value<string>
    {
        public StringValue(string value)
            : base(value) { }
    }
}

File: encoders/ql/src/main/java/org/uva/sea/ql/encoders/runtime/value/StringValue.java
package org.uva.sea.ql.encoders.runtime.value;

public class StringValue extends Value {

	private String value;

	public StringValue(String value) {
		this.value = value;
	}

	public void setValue(String value) {
		this.value = value;
	}

	public String getValue() {
		return value;
	}
}
